app/controllers/admin/users_controller.rb

```diff
@@ -15,8 +15,10 @@ def show
     if @user.current_sign_in_at
       @last_login = @user.current_sign_in_at.to_formatted_s(:short)
     else
-      @last_login = "never"
+      @last_login = 'never'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.